config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/util/entity/AnnotationConstants.java

```diff
@@ -7,6 +7,8 @@
 
     public static final String ANNOTATION_TYPE_BUNDLE = "@bundle";
     public static final String ANNOTATION_TYPE_REUSABLE = "@reusable";
+    public static final String ANNOTATION_TYPE_REUSABLE_BUNDLE = "@reusableBundle";
```

Comment:
Whats the difference between @reusable and @reusableBundle?

Reply:
both are same, raju asked me to keep reusable for sometime